core/Cargo.toml

```diff
@@ -151,6 +151,7 @@ unicase = { version = "2.7.0", optional = true }
 arbitrary = { version = "1.3.2", features = ["derive"], optional = true }
 regex-syntax = { version = "0.8.2", optional = true, features = ["arbitrary"] }
 geo-types = { version = "0.7.12", features = ["arbitrary"] }
+reblessive = { version = "0.1.1", path = "../../reblessive" }
```

Comment:
Just placing a comment to make sure this get switched to a crate before merged 🙏